docs/telemetry.md

```diff
@@ -38,7 +38,6 @@ For example, this is the configuration template for a local development env that
 ```bash
 ENABLE_OPEN_TELEMETRY=true
 
-OTEL_COLLECTOR_OTLP_HTTP_ENDPOINT="https://hostname.grafana.net/otlp"   # e.g. Grafana OTLP HTTP endpoint 
```

Comment:
@mj3cheun I think I may have lost this in the mere, and don't know what the right thing to do here, can you check in case a new PR needed to reintroduce?

(pushing updates in a few sec)